pkg/infra/tracing/tracing.go

```diff
@@ -20,13 +17,67 @@ import (
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
 	trace "go.opentelemetry.io/otel/trace"
+
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/setting"
 )
 
 const (
 	envJaegerAgentHost = "JAEGER_AGENT_HOST"
 	envJaegerAgentPort = "JAEGER_AGENT_PORT"
 )
 
+// Tracer defines the service used to create new spans.
+type Tracer interface {
```

Comment:
maybe we should rename this file to opentracing_tracing.go, I would leave the Tracer in the opentelemetry_tracer.go file, maybe would consider moving other stuff rather there

Reply:
@idafurjes We should probably split out all the OpenTracing specifics into one such file, these two interfaces are for both and provided by `ProvideService(…) Tracer` in the _tracing.go_-file, so I found it confusing to have them defined in the OTel file? 🤔

Reply:
or we deprecate opentracing

Reply:
🙂

Could we use OTel in the background for OpenTracing?

Reply:
even if we could, it would be painful

Reply:
:+1: Ok, let's not do that then